fossunited/templates/foss_base.html

```
@@ -33,21 +34,29 @@
+  {% block head_include %}
+    {{ super() }}
+    <link rel="stylesheet" href="/assets/fossunited/css/custom.css">
+  {% endblock %}
+  {% block page_style %}
+  {% endblock %}
+{% endblock %}
+{% block content %}
+<main id="main-content" class="base-placeholder" tabindex="-1">
+  {% block page_content %}
+    <h3>Your Content Goes Here.</h3>
+    <br />-> Change content inside "block page_content" in your page.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)